echopype/calibrate/calibrate_ek.py

```diff
@@ -130,22 +130,27 @@ class CalibrateEK60(CalibrateEK):
     def __init__(self, echodata, env_params, cal_params, **kwargs):
         super().__init__(echodata, env_params, cal_params)
 
-        # Set sonar_type
+        # Set sonar_type and waveform/encode mode
         self.sonar_type = "EK60"
```

Comment:
Small comment: we use `sonar_model` in the echodata class and elsewhere. If it's not too disruptive, I'd suggest calling it that here, too, to maintain consistency across echopype and reduce cognitive overload.

Reply:
Hmm, I used `sonar_type` because here I use "EK60" to stand for the older style of data, and "EK80" to stand for the newer style of data. Perhaps I can use "Ex60" (though this does not include ES70) and "Ex80" if we use `sonar_model`. Thoughts?

Reply:
Hmm, I don't know what "Ex60" and "Ex80" are, so I can't comment on that.

But this may be related to the broader relationship between Simrad-Kongsberg `sonar_model` entries (eg, "EK60", "ES70") and the broader sonar "type" ("EK60" vs "EK80") that determines the parser, set_group_, and many other processing decisions. I've created #995 just now to describe this relationship and a need for describing it centrally in an easily reusable way.

But back to the question at hand, *here.* No need to belabor my minor naming point. If you think `sonar_type` is more appropriate, stick with it. Or if you think this really is related to #995, stick with `sonar_type` and let's discuss the broader I raised in #995 issue after 0.7.0.

Reply:
"Ex60" and "Ex80" are what you called "class" in #995. I think we can keep it as is here, and address those code consolidation changes later in a more systematic way.

Reply:
sounds good